sbnana/SBNAna/Cuts/NueCuts.cxx

```diff
@@ -146,8 +147,9 @@ namespace ana{
   const Cut kNueAllTrackDazzleMuonScoreCut(
     [](const caf::SRSliceProxy* slc)
     {
-      for (auto const& trk : slc->reco.trk)
+      for (auto const& pfp : slc->reco.pfp)
```

Comment:
I think this is missing a check on the track score if we want to be completely backwards compatible